l10n_ch_dta/wizard/create_dta.py

```diff
@@ -724,10 +724,8 @@ def _create_dta(self, data):
                 record_type = RecordGt836
             elif country_code and country_code != 'CH':
                 record_type = RecordGt836
-            elif elec_pay == 'bvr':
+            elif elec_pay == 'postal':
```

Comment:
More precisely you can check the field `communication_type` on `account.payment.line`